src/platforms/x11/graphics/display.h

```diff
@@ -104,17 +106,19 @@ class Display : public graphics::Display
     Frame last_frame_on(unsigned output_id) const override;
 
 private:
-    struct OutputInfo
+    struct OutputInfo : ::mir::X::X11Resources::VirtualOutput
```

Comment:
It seems unintuitive for a `struct` to implement an interface. OTOK I guess this is "private" to `Display` and the only reason the struct is declared in the header is to preced the use with `unique_ptr`.